generate fails on Windows whenever enums are used in schema

**Description and expected behavior**

`zenstack generate` command fails _on Windows_ with the following error whenever enums are used in the schema.zmodel file. Remove the enums and the generate command works successfully.

```
⌛️ ZenStack CLI v1.0.0-beta.8, running plugins
✔ Running plugin Prisma
✔ Running plugin Model Metadata
⠋ Running plugin Access PolicyError compiling generated code:
node_modules/.zenstack/policy.ts:3:22 - error TS2307: Cannot find module '....srcgeneratedprisma-client' or its corresponding type declarations.

3 import { role } from "..\..\src\generated\prisma-client";
                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

✖ Running plugin Access Policy
: Error compiling generated code
```

Reproduction repo here: https://github.com/benrutt99/zenstack-test


**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment (please complete the following information):**

-   ZenStack version: 1.0.0-beta.8
-   Prisma version: 4.14.0
-   Database type: Postgresql

**Additional context**

* this issue only seems to occur on Windows. On Mac this issue does not occur.

--- a/packages/sdk/src/prisma.ts
+++ b/packages/sdk/src/prisma.ts
@@ -54,5 +54,9 @@ export function getPrismaClientImportSpec(model: Model, importingFromDir: string
     // console.log('IMPORTING PATH:', importingFromDir);
 
     // compute prisma client absolute output dir relative to the importing file
-    return path.relative(importingFromDir, resolvedPrismaClientOutput);
+    return normalizePath(path.relative(importingFromDir, resolvedPrismaClientOutput));
+}
+
+function normalizePath(p: string) {
+    return p ? p.split(path.sep).join(path.posix.sep) : p;
 }

